test/index.js

```diff
@@ -77,6 +76,7 @@ async function createMember(){
       is_admin_invited: 0,
     });
     console.log(member);
+    console.log(member.getGroup());
```

Comment:
testing sequelize [mixins](https://sequelize.org/master/manual/assocs.html#special-methods-mixins-added-to-instances)